File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/settings/account/AccountSettingsEvent.kt
package com.hhp227.concafe.presentation.settings.account

sealed interface AccountSettingsEvent {
    data object NavigateBack : AccountSettingsEvent
    data class NavigateToCastEdit(val cafeId: String?, val castId: String?) : AccountSettingsEvent
    data object NavigateToChangePassword : AccountSettingsEvent
    data class ShowMessage(val message: String) : AccountSettingsEvent
}

File: iosApp/iosApp/Presentation/Settings/Account/AccountSettingsEvent.swift
//
//  AccountSettingsEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/14.
//

import Foundation

enum AccountSettingsEvent {
    case navigateBack
    case navigateToCastEdit(cafeId: String?, castId: String?)
    case navigateToChangePassword
    case showMessage(String)
}
